src/components/Practice/Mechanics5.jsx

```
@@ -180,285 +178,305 @@ const Mechanics5 = ({
-                {playingIndex === "question" ? (
-                  <StopAudioButton size={35} color={"#1CB0F6"} />
-                ) : (
-                  <PlayAudioButton size={35} color={"#1CB0F6"} />
-                )}
+                <audio
+                  key={question_audio} // Key added to force remount when source changes
+                  ref={questionAudioRef}
+                  preload="metadata"
+                  onPlaying={() => setPlayingIndex("question")}
+                  onPause={() => setPlayingIndex(null)}
+                >
```

ESLint

[error] Media elements such as <audio> and <video> must have a <track> for captions.

(jsx-a11y/media-has-caption)